### DISCUSSION TITLE
Whoogle and Google cookie consent fix

### DISCUSSION THREAD

**Discussion by AdmiralAwesome** [tteck/Proxmox#1708] [General]
I've found a solution to Google Cookie consent error when running Whoogle from script in LXC.

Stop Whoogle service:
`systemctl stop whoogle.service`

`nano /usr/local/lib/python3.9/dist-packages/app/request.py`
Ctrl+W, find `cookies`

Then replaced the complete 'cookies block' for:

```
        # view is suppressed correctly
        now = datetime.now()
        cookies = {
            'CONSENT': 'PENDING+987',
            'SOCS': 'CAESHAgBEhJnd3NfMjAyMzA4MTAtMF9SQzIaAmRlIAEaBgiAo_CmBg'
        }
```
![image](https://github.com/tteck/Proxmox/assets/2470577/cb9e1185-8edc-4e66-8b66-35356f0404cf)

Start Whoogle service:
`systemctl start whoogle.service`

Check Whoogle service status:
`systemctl status whoogle.service`

I've found this solution here: https://github.com/benbusby/whoogle-search/issues/1053#issuecomment-1684062876 so be sure to drop a like to original author. 👍

**Comment by Smile2Grow** [comment]
It works! Thank you.

**Reply by tteck** [reply]
Is it just an indention issue?

**Reply by Smile2Grow** [reply]
No, the code described above for cookies is actually different.

**Reply by Smile2Grow** [reply]
There is also an update from Whoogle on Github.
